decidim-accountability/spec/shared/manage_results_examples.rb

```diff
@@ -30,7 +30,7 @@
     end
   end
 
-  context "previewing results" do
+  context "previewing results", driver: :poltergeist do
     it "allows the user to preview the result" do
       within find("tr", text: translated(result.title)) do
         @new_window = window_opened_by { find("a.action-icon--preview").click }
```

Comment:
I think we could change this test by asserting its `href` and `target` attributes instead and totally ditch the `poltergeist` dependency.